bokeh/models/annotations/html/tooltips.py

```diff
@@ -0,0 +1,77 @@
+#-----------------------------------------------------------------------------
+# Copyright (c) 2012 - 2021, Anaconda, Inc., and Bokeh Contributors.
+# All rights reserved.
+#
+# The full license is in the file LICENSE.txt, distributed with this software.
+#-----------------------------------------------------------------------------
+'''
+
+'''
+#-----------------------------------------------------------------------------
+# Boilerplate
+#-----------------------------------------------------------------------------
+from __future__ import annotations
+
+import logging # isort:skip
+log = logging.getLogger(__name__)
+
+#-----------------------------------------------------------------------------
+# Imports
+#-----------------------------------------------------------------------------
+
+# Bokeh imports
+from ....core.enums import TooltipAttachment
+from ....core.properties import Bool, Enum, Override
+from .html_annotation import HTMLAnnotation
+
+#-----------------------------------------------------------------------------
+# Globals and constants
+#-----------------------------------------------------------------------------
+
+__all__ = (
+    "Tooltip",
+)
+
+#-----------------------------------------------------------------------------
+# General API
+#-----------------------------------------------------------------------------
+
+class Tooltip(HTMLAnnotation):
```

Comment:
aside: we could/should rename this for 3.0 and make it user-usable (e.g. the "InfoPane" work, but I do hate that name)

Reply:
My intention is to restart work on persistent tooltips, so I suppose this can be done then.